pysmhi/smhi.py

```diff
@@ -46,7 +46,8 @@ async def async_get_data(
                 data: dict[str, Any] = await resp.json()
 
         except Exception as error:
-            LOGGER.debug("Error, status: %s, error: %s", resp.status, str(error))
+            status = resp.status if resp else None
```

Comment:
I don't think this fixes it. The `resp` variable may not be defined if the `self._session.get` call raises.

Reply:
You're right, it went to quick. Fixing that now properly.

Reply:
#73 to fix

Reply:
Thanks!